client-runtime/src/main/java/com/microsoft/rest/v2/http/NettyClient.java

```diff
@@ -108,7 +176,12 @@ public void channelCreated(Channel ch) throws Exception {
             }, channelPoolSize);
         }
 
-        private Single<HttpResponse> sendRequestInternalAsync(final HttpRequest request, Proxy proxy) {
+        private boolean useZeroCopy(Channel channel) {
+            boolean isSSL = channel.pipeline().get(SslHandler.class) != null;
+            return !isSSL && !(eventLoopGroup instanceof NioEventLoopGroup);
```

Comment:
Can I use an `OioEventLoopGroup`?

Reply:
Ah, yeah--this is really trying to say "if it's a native transport", but if we for some reason moved over to an OioEventLoopGroup, this might do something bad.

Do you think I should check the class's name against a string instead?

Reply:
The only 2 we are using are Epoll and KQueue? Can we check if we are using them?

Reply:
Since the classes are not present at compile time, it would be necessary to check with the fully qualified class name, e.g. "io.netty.channel.epoll.EpollEventLoopGroup"

Reply:
You are right. I'd prefer that considering there are many event loop group implementations: https://netty.io/4.0/api/io/netty/channel/EventLoopGroup.html